functions.php

```diff
@@ -68,6 +68,47 @@ function course_theme_init() {
 			'label' => __( 'Button', 'course' ),
 		)
 	);
+	register_block_style(
+		'core/group',
+		array(
+			'name'  => 'group-left-border',
+			'label' => __( 'Left Border', 'course' ),
+		)
+	);
+	register_block_style(
+		'core/image',
+		array(
+			'name'  => 'course-image-border',
+			'label' => __( 'Image Border', 'course' ),
+		)
+	);
+
+	$block_pattern_categories = array(
+		'sensei-lms' => array( 'label' => __( 'Sensei LMS', 'course' ) )
+	);
+
+	/**
+	 * Filters the theme block pattern categories.
+	 *
+	 * @since Course 1.0
+	 *
+	 * @param array[] $block_pattern_categories {
+	 *     An associative array of block pattern categories, keyed by category name.
+	 *
+	 *     @type array[] $properties {
+	 *         An array of block category properties.
+	 *
+	 *         @type string $label A human-readable label for the pattern category.
+	 *     }
+	 * }
+	 */
+	$block_pattern_categories = apply_filters( 'course_block_pattern_categories', $block_pattern_categories );
+
+	foreach ( $block_pattern_categories as $name => $properties ) {
```

Comment:
Could you please provide more context about why we need to use PHP only to register the pattern?
it theoretically should be automatically, rigth?

Reply:
Thanks @gabrielcaires , it is update here https://github.com/Automattic/sensei-theme/pull/44/commits/73331765814cfc083618f8a463f16acd60d6cde1